about.html

```diff
@@ -12,53 +12,64 @@
 </head>
 
 <body>
+  <div class="container">
+  <div class="navigation-bar">
+    <div class="lambda-logo">
+      <h1>ambda</h1>
+    </div>
+    <div class="nav-bar">
```

Comment:
You should also put a link back to index (Home) so the user can get back to the home page.